stable/dex/values.yaml

```diff
@@ -0,0 +1,100 @@
+# Default values for dex
+# This is a YAML-formatted file.
+# Declare name/value pairs to be passed into your templates.
+# name: value
+
+image: quay.io/coreos/dex
+imageTag: "v2.10.0"
+imagePullPolicy: "IfNotPresent"
+
+inMiniKube: false
+
+nodeSelector: {}
+
+externalIPs: []
+
+replicas: 1
+
+
+service:
+  type: ClusterIP
+  port: 5556
+  # externalIPs:
+
+resources:
```

Comment:
We do not recommend setting default resources unless there are very good reasons to do so.

Reply:
fixed